### PR TITLE
Assorted docs fixes

### PR DESCRIPTION
**Pull request by muffinresearch**
* Use rtd theme for local docs builds
* Build docs on make loop
* Add docs for docker-utils

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -57,6 +57,12 @@ on your host machine::
     make initialize_docker  # Answer yes, and create your superuser when asked.
     # Once it's all loaded, go to http://localhost:8000 and enjoy!
 
+.. note::
+    docker-utils_ is a wrapper library written in Python that wraps the
+    docker-compose command. It add sugar for things like getting a bash shell
```
Comment (magopian) [inline docs/index.rst]: s/It add/It adds/ ?

File: docs/conf.py [hunk 2]
```diff
@@ -98,6 +98,13 @@
 #html_theme_path = ['_themes']
 #html_theme = 'mozilla'
 
+on_rtd = os.environ.get('READTHEDOCS', None) == 'True'
```
Comment (magopian) [inline docs/conf.py]: in https://pypi.python.org/pypi/sphinx_rtd_theme/0.1.8#using-this-theme-locally-then-building-on-read-the-docs there's a little comment above this line which would make sense to copy here

File: requirements/docs.txt [hunk 3]
```diff
@@ -3,4 +3,4 @@ sphinxcontrib-httpdomain==1.1.9
 docutils==0.11
 Pygments==1.6
 Sphinx==1.2.2
-mdn-sphinx-theme>=0.3
+sphinx_rtd_theme==0.1.8
```
Comment (magopian) [inline requirements/docs.txt]: There's a 0.1.9 version, should we use that instead?

**Comment by magopian** [comment]
Excellent, thanks!

r+wc

File: requirements/docs.txt (hunk 3 above)
Comment (muffinresearch) [inline requirements/docs.txt]: Yeah, the tests blew up with that version. I need to learn what the process is dealing with python packages is - presumably it's not on pyrepo?

Comment (magopian) [inline requirements/docs.txt]: indeed, it's not on https://pyrepo.addons.mozilla.org/
I'll add it straight away though

**Pull request merged**